src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/idam/AuthenticationCheckerTest.java

```diff
@@ -156,7 +156,7 @@ private FeignException createFeignException(int httpStatus) {
             .errorStatus("method1", Response
                 .builder()
                 .request(mock(Request.class))
-                .body(new byte[0])
+                .body("Error response from IDAM".getBytes())
```

Comment:
where is 'Error response from IDAM' coming from?

Reply:
It's just an example response. I'm giving it some value here to check if whatever's here ends up in the response of the IDAM status endpoint.

Reply:
oops, it's setup stage, thought it was verification